tests/integration/strategy_tests.cpp

```diff
@@ -389,11 +389,43 @@ TEST(ReleaseTest, Works)
   void* ptr_two = alloc.allocate(1024);
   alloc.deallocate(ptr_two);
 
+  ASSERT_EQ(alloc.getCurrentSize(), 62);
   EXPECT_NO_THROW(alloc.release());
 
+  ASSERT_LE(alloc.getActualSize(), 64);
+
```

Comment:
Why isn't this failing?  Should `alloc.getActualSize() == (1024+64)`?

Reply:
No, `release` is called, and ptr_two has been deallocated.

Reply:
Oh duh.  I missed the release() call.  I suppose we could consider adding another ASSERT_EQ before the release for getActualSize()?